docs/install.md

```diff
@@ -0,0 +1,150 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Instructions to Build and Install Private Data Objects
+
+Private Data Objects (PDO) can be installed directly to a host system or
+it may be installed through Docker. This document contains instructions
+for both methods of installation.
+
+PDO requires installation of the SDK for Intel Software Guard Extensions
+(SGX). SGX can be used either in simulation mode or hardware
+mode. Simulation mode will work even when there is no processor support
+for SGX. However, simulation mode is not secure and is suitable soley
+for development and demonstration. Hardware mode, which uses trusted
+execution capabilities in the processor to protect confidentiality and
+integrity of computation, requires appropriate processor
+support. Information about supported processors is provided below.
+
+## Recommended Host System
+
+The recommended host-system configuration for Private Data Objects is to
+separate the Private Data Objects components from the Sawtooth components. This
+means (at least) two different physical systems if using SGX-enabled hardware.
```

Comment:
This seems to be the first reference to Sawooth. I think we could explain what it is and how PDO depends on it.